### ISSUE TITLE
"Package does not exist" when running runtest.bat on  A-TextUiTesting Level

### ISSUE THREAD

**Issue by eugenetyc** [nus-cs2103-AY1920S2/forum#9]
Hi guys, does anyone know why is the IDE showing that the packages do not exist? As you can see, they are available in the left panel of the IDE (ie Driver, Tasks, Utilities, WelcomeGoodbye).

I have already updated my PATH to be of Java 12 SDK, so the issues of javac don't really occur to me.

Any help is greatly appreciated!

![Capture](https://user-images.githubusercontent.com/40591060/72898776-ae14af00-3d5f-11ea-9ae6-54c1debcca56.JPG)

**Comment by mario7lorenzo** [comment]
Hi there! I just want to make sure whether the packaged imports have been compiled or not? Because if you have not compile the packages yet, it will throw such errors. So try check and whether there exists the FileName.class file in the Utilities directory. Notice that in your shell script, you only compile your Duke class!

**Comment by eugenetyc** [comment]
Dear Mario,

Thanks for the help, but it still isn't working. I refactored the classes, moving them out of their initial folders. Then, compiled them with javac -d . *.java to obtain the following.

![Capture](https://user-images.githubusercontent.com/40591060/72902979-5e39e600-3d67-11ea-973d-4ec7bb38359d.JPG)

As it seems, the issue of packages do not exist still occur. :/

**Comment by eugenetyc** [comment]
Hi guys! Alright, so here's a breakdown of what I did, hopefully it will simplify things for your guys. Thanks @mario7lorenzo for the help, I reread what you explained, and I've come up with the following points:

1. If you've not put the classes in packages, simply find the directory containing all the java files and do a javac *.java, rather than just compiling javac Duke.java.
2. There is no need to create a file called ACTUAL.txt. The reason being output redirection will create the file if it is not present, and the script deletes any existing files called ACTUAL.txt in the directory.

While I got gotten the classes and hence the script (.bat file) to work with the above without packages, do let me know if you guys figure out a way with packages, if we aren't gonna be taught in the later weeks!

Cheers!
